h/oauth/jwt_grant_token.py

```diff
@@ -26,7 +26,10 @@ def __init__(self, token):
         self._token = token
 
         try:
-            self._claims = jwt.decode(token, verify=False)
+            self._claims = jwt.decode(
```

Comment:
https://pyjwt.readthedocs.io/en/stable/changelog.html#dropped-deprecated-verify-param-in-jwt-decode